Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM ruby:2.4.1-jessie
+
+RUN cat /etc/os-release
```

Comment:
Jeżeli jest zbędne, bym to wyrzucił.

Reply:
Tak, jest zbędne nie zauważyłem że zostało.